draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -145,6 +155,20 @@ const TooltipContent = ({
   ) : null
 }
 
+const renderChildren = (
+  content: ReactNode,
+  tooltipId: string,
+  hasActiveTooltip: boolean
+): ReactNode => {
+  if (isSemanticElement(content)) {
```

Comment:
Idea: we could throw a console warning or something like that if the element passed in here is `isSemanticElement: false`. People don't love/respond well to the Icon warnings we have, but that's probably more because it could have been enforced with TS instead. Might be better than nothing.

Reply:
I have no opposition there, might help be a quick way to identify it for them if they're digging in console 👍